test/e2e/app-dir/metadata-dynamic-routes/app/twitter-image.tsx

```diff
@@ -1,4 +1,14 @@
 import { ImageResponse } from 'next/server'
+// @ts-ignore
+import { ImageResponse as ImageResponse2 } from '@vercel/og'
+
+// Node.js: Using @vercel/og external package, and should be aliased to "next/server" ImageResponse
+if (ImageResponse.displayName !== 'NextImageResponse')
+  throw new Error('ImageResponse mismatch: ' + ImageResponse.displayName)
+// @ts-ignore
+if (ImageResponse2.displayName !== 'NextImageResponse')
+  // @ts-ignore
+  throw new Error('ImageResponse mismatch: ' + ImageResponse2.displayName)
```

Comment:
I guess we can just test with `ImageResponse === ImageResponse2` instead of `displayName`?

Reply:
No there's an extra step of wrapping ImageResponse in precompiled one, that's why I added `displayName`

Reply:
Oh so you meant that even with this fix, they will still be different instances?